qutebrowser/browser/commands.py

```diff
@@ -1748,3 +1756,19 @@ def fullscreen(self, leave=False, enter=False):
 
         log.misc.debug('state before fullscreen: {}'.format(
             debug.qflags_key(Qt, window.state_before_fullscreen)))
+
+    def possibly_open_in_existing_tab(self, url):
+        if not config.val.tabs.switch_to_open_url:
+            return False
+
+        tab = next((t for t in tabsfinder.all_tabs() if t.url == url), None)
+
+        if tab is not None:
+            index = "{}/{}".format(tab.win_id, tab.index)
+            self.buffer(index)
+            message.info("URL is open already. Switching to it. {}".
+                         format(index))
```

Comment:
This seems like an odd message to show ("URL is open already. Switching to it. 02/15"). Is it necessary to show it at all? If so, should the index be included?

Reply:
Removed.